bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ElectionRecord/ElectionRecordManager.cs

```diff
@@ -0,0 +1,306 @@
+using System.IO.Compression;
+using ElectionGuard.Converters;
+using ElectionGuard.Decryption.Tally;
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.ElectionRecord;
+
+public static class ElectionRecordManager
+{
+    public static readonly string GUARDIAN_FOLDER = "guardians";
```

Comment:
all of this was in the ElectionRecordGenerator in the ui project but moved here so it is available in the cli tool as well